hello-ethernet/prj.conf

```diff
@@ -0,0 +1,29 @@
+# Generic networking options
+CONFIG_NETWORKING=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_IPV6=n
+
+# Logging
+CONFIG_NET_LOG=y
+
+# Network Shell
+CONFIG_NET_SHELL=y
+
+# TLS configuration
+CONFIG_MBEDTLS_ENABLE_HEAP=y
+CONFIG_MBEDTLS_HEAP_SIZE=10240
+CONFIG_MBEDTLS_SSL_MAX_CONTENT_LEN=2048
+
+# Application
+CONFIG_MAIN_STACK_SIZE=4096
+
+CONFIG_GOLIOTH=y
+CONFIG_GOLIOTH_SYSTEM_CLIENT=y
+
+CONFIG_MINIMAL_LIBC_MALLOC_ARENA_SIZE=256
+
+CONFIG_LOG_BACKEND_GOLIOTH=y
+CONFIG_LOG_PROCESS_THREAD_STACK_SIZE=2048
+
+CONFIG_GOLIOTH_SYSTEM_CLIENT_PSK_ID="stm32f405-feather-id@ttgo-demo"
+CONFIG_GOLIOTH_SYSTEM_CLIENT_PSK="cfa0c9537863f6268d86088376e62e0b"
```

Comment:
Credential leaking here

Reply:
Good catch, I've removed this code and deleted the credentials from the console.